gnmi_cli_py/py_gnmicli.py

```diff
@@ -430,8 +440,14 @@ def subscribe_start(stub, options, req_iterator):
               print('gNMI Error '+str(response.error.code)+\
                 ' received\n'+str(response.error.message) + str(response.error))
           elif response.HasField('update'):
-              print(response)
-              update_count = update_count+1
+              if filter_event_regex is not None and filter_event_regex is not "":
+                  match = check_event_response(response, filter_event_regex)
+                  if len(match) is not 0:
+                      print("{} was found in {}".format(match,response))
+                      update_count = update_count + 1
+              else:
```

Comment:
>else [](http://example.com/codeflow?start=14&length=4)

report an error if `filter_event_regex` is empty.

Reply:
done